README.md

```diff
@@ -85,19 +85,24 @@ dependencies for a given platform.
 ### Setting up dependencies
 
 The following setup scripts use the `DEPENDENCY_DIR` environment variable to set the
-location of the build packages. If you do not set this variable, it will default to
-the current working directory.
-
-```shell
-$ export DEPENDENCY_DIR=/path/to/your/dependencies
-```
+location to download and build packages. This defaults to `deps-download` in the current
+working directory. Use `INSTALL_PREFIX` to set the install directory of the packages.
```

Comment:
nit: could we start `INSTALL_PREFIX` env in a new paragraph for clarity?

Reply:
Done!